src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -162,6 +162,16 @@
                 <muxc:NavigationViewItem.Icon>
                     <FontIcon Glyph="&#xE713;" />
                 </muxc:NavigationViewItem.Icon>
+                <muxc:NavigationViewItem.ContextFlyout>
+                    <MenuFlyout>
+                        <MenuFlyoutItem
+                                        x:Name="OpenJsonFolderNavItem"

```

Comment:
If you give this a `x:Uid`, like:
```xaml
            <muxc:NavigationViewItem x:Name="OpenJsonFolderNavItem"
                                     x:Uid="Nav_OpenJsonFolder"
```

Then you should be able to set the text in the `TerminalSettingsEditor\Resources\en-us\resources.resw` like:

```xml
  <data name="Nav_OpenJsonFolder.Text" xml:space="preserve">
    <value>Open JSON folder</value>
  </data>
```

And even the tooltip should be settable with
```xml
  <data name="Nav_OpenJsonFolder.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
    <value>Open the folder holding your settings.json file.</value>
  </data>
```

And if that doesn't work, then there's something wacky going on with XAML

Reply:
Hmm... Mind committing your changes anyways? I tried the suggested changes on my end, and it worked fine. I'd be happy to take a closer look at your approach and let you know how to fix it :)

Also, you're not referencing the name `OpenJsonFolderNavItem` anywhere, so you might as well remove it.

Reply:
It's a relic of me trying to get the resources to work! Curious that it works on your end. Must be my system. I'll commit the changes, maybe I am doing something wrong, too. I appreciate the feedback!